run.sh

```diff
@@ -22,9 +22,15 @@ s3get () {
 
     mkdir -p /s3-archives
 
-    aws s3 cp s3://${AWS_BUCKET}/${REMOTE_FILE}  ${DEST}/${REMOTE_FILE}
-
-    if [[ $? != 0 ]]; then  fail "${REMOTE_FILE} doesn't exist in s3"; fi;
+    result=$(aws s3 cp s3://${AWS_BUCKET}/${REMOTE_FILE}  ${DEST}/${REMOTE_FILE} 2>&1)
+
+    if [[ $? != 0  && $result == *"does not exist"* ]]; then
+      echo $result
+      FALL_BACK_FILE="edward-master.tar.gz"
+      aws s3 cp s3://${AWS_BUCKET}/${FALL_BACK_FILE}  ${DEST}/${FALL_BACK_FILE}
+    elif [[ $? != 0 ]];then
+      fail $result
+    fi
```

Comment:
this should only happen when the container first runs, not everytime it fails to pull edward (else a glitch on s3 might make us redeploy master on edward without knowing)